crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -22,6 +44,7 @@ pub struct ModuleOptionsContext {
     pub enable_styled_components: bool,
     pub enable_styled_jsx: bool,
     pub enable_postcss_transform: Option<PostCssTransformOptions>,
+    pub enable_webpack_loaders: Option<WebpackLoadersOptions>,
```

Comment:
It seems weird that this is worded as a boolean (`enable_something`) instead of just `webpack_loaders`, like we have for other options. Same comment goes for PostCSS.

Reply:
yeah. `pub enable_styled_jsx: bool,` could also be `pub styled_jsx: bool,`